portable/Common/mpu_wrappers.c

```diff
@@ -91,7 +91,7 @@
     #if ( configSUPPORT_STATIC_ALLOCATION == 1 )
         TaskHandle_t MPU_xTaskCreateStatic( TaskFunction_t pxTaskCode,
                                             const char * const pcName,
-                                            const uint32_t ulStackDepth,
+                                            configSTACK_DEPTH_TYPE uxStackDepth,
```

Comment:
``const`` keyword has been removed, it was present in in the initial parameter list for uxStackDepth

Reply:
Fixed.

Also adjusted `MPU_xTaskCreate()` so that it follows its prototypes too.